integration-tests/api-gateway/abandon/abandon-happy.test.ts

```diff
@@ -66,7 +72,10 @@ describe("Given the session is valid and expecting to abandon the journey", () =
   });
 
   it("Should receive a 200 response when /abandon endpoint is called without optional headers", async () => {
-    const abandonResponse = await abandonEndpoint({ "session-id": sessionId });
+    const privateApi = `${output.PrivateApiGatewayId}`;
```

Comment:
Should we just reference the `privateApi` that is being defined in the `beforeEach`?